PresentScreenings/AppDelegate.designer.cs

```diff
@@ -20,6 +20,9 @@ partial class AppDelegate
 		[Outlet]
 		AppKit.NSMenuItem _combineTitlesMenuItem { get; set; }
 
+		[Outlet]
+		AppKit.NSMenuItem _downloadFilmInfoMenuItem { get; set; }
+
```

Comment:
Please consider to get this out of Xcode, because using it from the C# code will crash the application.

Reply:
Done.